pom.xml

```diff
@@ -138,7 +138,7 @@ under the License.
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-archiver</artifactId>
-      <version>3.6.0</version>
+      <version>3.6.1-SNAPSHOT</version>
```

Comment:
Please don't use SNAPSHOT versions, as this could introduce instability at any given time.

Reply:
Hi @spiritualops , yes of course, I'm just waiting plexus release (maybe it's already done) to change this with the correct release version

And I need help to maybe add some test, and for MDO

Thx

Reply:
The Plexus Archiver has been released and the assembly plugin is already updated to use a newer version so you can just rebase your branch.